.travis.yml

```diff
@@ -11,10 +11,7 @@ script:
   - bundle exec jekyll build
   - percy snapshot _site/
   - ruby ./scripts/image_checker.rb
-  - docker run --volume=$(pwd):/app --workdir=/app coala/base:pre /bin/bash -c
-    "coala --ci --apply-patches --no-orig;
-    coala-ci || true"
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/coala_patch.sh; fi'
```

Comment:
Read the commit message
https://github.com/fossasia/gci16.fossasia.org/pull/656/commits/a9a518148de424626d5b96c4c03b5fe4e37668c3